Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4029,3 +4047,114 @@ def update_integration_context(context, object_keys=None, sync=True):
 
 class DemistoException(Exception):
     pass
+
+
+def arg_to_timestamp(arg: Any, arg_name: str, required: bool = False) -> int:
+    """Converts an XSOAR argument to a timestamp (seconds from epoch)
+
+    This function is used to quickly validate an argument provided to XSOAR
+    via ``demisto.args()`` into an ``int`` containing a timestamp (seconds
+    since epoch). It will throw a ValueError if the input is invalid.
+    If the input is None, it will throw a ValueError if required is ``True``,
+    or ``None`` if required is ``False.
+
+    :type arg: ``Any``
+    :param arg: argument to convert
+
+    :type arg_name: ``str``
+    :param arg_name: argument name
+
+    :type required: ``bool``
+    :param required:
+        throws exception if ``True`` and argument provided is None
+
+    :return:
+        returns an ``int`` containing a timestamp (seconds from epoch) if conversion works
+        returns ``-1`` if arg is ``None`` and required is set to ``False``
+        otherwise throws an Exception
+    :rtype: ``Optional[int]``
+    """
+
+    if arg is None:
+        if required is True:
+            raise ValueError(f'Missing "{arg_name}"')
+        return -1
+
+    if isinstance(arg, str) and arg.isdigit():
```

Comment:
if arg contains a millisecond epoch timestamp - this function will return it unchanged but it needs to be converted to seconds for consistency. Same for the "int/float" section lines 4096-4098